.github/workflows/cleanup.yml

```diff
@@ -0,0 +1,126 @@
+name: Bulk Repo Update
+
+on:
+  workflow_dispatch:
+    inputs:
+      organization:
+        description: "Github organization for the listed repos"
+        type: choice
+        required: true
+        default: openedx
+        options:
+         - openedx
+         - edx
+      repos_list:
+        description: "List of repositories in the format 'x', 'y', 'z'..."
+        type: string
+        required: true
+      python_version:
+        description: "Python Version"
+        type: choice
+        required: true
+        default: '3.8'
+        options:
+         - '3.8'
+      packages:
+        description: "Space separated list of required packages"
+        type: string
+        required: false
+        default: ''
+      commit_message:
+        description: "Commit message/ PR title"
+        type: string
+        required: true
+      pr_body:
+        description: "Additional information for PR body"
+        type: string
+        required: false
+        default: ''
+      branch:
+        description: "Branch name"
+        type: string
+        required: true
+        default: 'update-code'
+      draft:
+        description: "Create draft PR"
+        type: boolean
+        required: false
+        default: false
+      force_delete_old_prs:
+        description: "Forcefully delete old PRs"
+        type: boolean
+        required: false
+        default: false
+      script:
+        description: "Script"
+        type: string
+        required: true
+
+jobs:
+
+  repos_list:
+    runs-on: ubuntu-20.04
+
+    outputs:
+      output1: ${{ steps.repos_list.outputs.list }}
+    steps:
+
+    - name: get repos list
+      id: repos_list
+      run: |
+        echo "::set-output name=list::[${{github.event.inputs.repos_list}}]"
+
+  bulk_update:
+    runs-on: ubuntu-20.04
+    needs: [ repos_list ]
+    strategy:
+      fail-fast: false
+      matrix:
+        repos: ${{fromJson(needs.repos_list.outputs.output1)}}
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+            repository: ${{ github.event.inputs.organization}}/${{ matrix.repos }}
+            ref: 'master'
```

Comment:
Some repos have different branch for active development e.g. `django-wiki` uses the `edx_release` for active development. Can we add an input for the ref branch? or mention in the description that this job will not work if your active development branch is not `master`.
This will also be useful in near future when the repos start switching from `master` branch to `main` as per the general community trend.